modules/enabling-scanner-v4-after-operator-installation.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-after-operator-installation_{context}"]
+= Enabling RHACS Scanner V4 after Operator installation
+
+[role="_abstract"]
+Scanner V4 is not enabled by default. If you did not enable it during installation, you can enable it after installation.
+
+First, you must enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. In the console, select *Operators* -> *Installed Operators* and select the {product-title-short} Operator.
+. Click *Central* in the menu bar.
+. Click *stackrox-central-services* to view the *stackrox-central-services* page, if you installed {product-title-short} using the default values.
```

Comment:
I'm not sure I'm following here, so apologies if I'm misunderstanding, but is "if you installed {product-title-short} using the default values." a condition for doing this step? If so, then this is a nit but per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#indicating-optional-and-conditional-steps) this condition should be located before the instruction:
```suggestion
. If you installed {product-title-short} using the default values, click *stackrox-central-services* to view the *stackrox-central-services* page.
```
That said, if there is a different step to take for users who didn't use default values, should that also be documented in this procedure?

Reply:
If you installed it using the default values, then the name of the cluster will be `stackrox-central-services`, but if you used your own values, it can be whatever you entered. I tried to rewrite to make that more clear - if you can PTAL :)